crates/daphne-server/src/roles/aggregator.rs

```diff
@@ -316,15 +316,33 @@ impl DapAggregator for crate::App {
 
         futures::stream::iter(agg_span)
             .map(|bucket| async move {
-                Ok::<bool, DapError>(
-                    !self
-                        .durable()
-                        .request(bindings::AggregateStore::Get, (version, task_id, &bucket))
+                let durable = self.durable();
+                let params = (version, task_id, &bucket);
+
+                let new_get_report_count = || {
+                    durable
+                        .request(bindings::AggregateStore::ReportCount, params)
+                        .send::<u64>()
+                };
+
+                let old_get_report_count = || {
+                    durable
```

Comment:
What do you mean by "old" and "new" here?

Reply:
it's the old way to do it and the new way to do it. Just because I want the daphne-server to be compatible with the previous version of the worker while things transition. Do you have a better name of these variables? I honestly don't like what I came up with

Reply:
Names are fine, I was not clear what the behavior was. Please add some documentation here to note that the old method is for backwards compatibility and will be removed after some future deploymenet chnage.